woocommerce/woocommerce-android
# Issue/2175 sale price validation

Fixes #2175 by adding logic to display an error message when the sale price entered is greater than the regular price.

#### Screenshots
LEFT: Before. RIGHT: After
<img src="https://user-images.githubusercontent.com/3903757/78157563-1c37bb00-740e-11ea-84ab-18ea28dd56ee.gif" width="300" />. <img src="https://user-images.githubusercontent.com/22608780/78233428-5a5ada80-74f3-11ea-8a01-00fdbaaa981a.gif" width="300" />

#### To test
- Click on a product from the products tab.
- Click on the Pricing section.
- Enter a sale price that is greater than the regular price.
- Notice an error message is displayed (same as what the web currently displays).

Update release notes:
